src/gluonts/model/deepar/_estimator.py

```diff
@@ -146,8 +147,8 @@ def __init__(
         assert num_layers > 0, "The value of `num_layers` should be > 0"
         assert num_cells > 0, "The value of `num_cells` should be > 0"
         assert dropout_rate >= 0, "The value of `dropout_rate` should be >= 0"
-        assert (cardinality is not None and use_feat_static_cat) or (
-            cardinality is None and not use_feat_static_cat
+        assert (cardinality and use_feat_static_cat) or (
+            not (cardinality or use_feat_static_cat)
```

Comment:
I think it's ok to set `cardinality`, even when `use_feat_static_cat` is `False`.

Then, I can deselect categorical features, by only setting `use_feat_static_cat`, and `cardinality` is still derived from the auto_fields.

Reply:
Yeah, good Idea.